### PR TITLE
adding Plantower PMS7003/5003/3003/1003 particle matter sensors

### PR DESCRIPTION
**Pull request by Adorfer**
Probably the sorting is not correct, i do not see where "P0" is matched towards PM01 and PM1 to PM25 plus P2 to PM10

### CODE REVIEW

**Comment by Adorfer** [comment]
this should do the job now

**Comment by ubergesundheit** [comment]
After reading up on the sensors, it appeared to me that the PMS7003/5003/3003/1003 are all separate models?

The way you've added the definition now is that the sensor models are not distinguishable. Could you add for each sensor a definition?

**Comment by Adorfer** [comment]
In terms of Firmware/adressing they are the same, just the 3003 is having a bit different datagram, which is handled by the driver.
Do you think it is worth, blowing up the list of options so much? I could do, but if I take all possible combinations with b

**Comment by ubergesundheit** [comment]
The sensor models should be distinguishable for data consumers. If you could the sensor models and model configurations I would be grateful.

For the pull request on the front end, maybe we could come up with a solution with dropdowns or something else.

**Comment by Adorfer** [comment]
PR modified as requested by @ubergesundheit 
(Please check for typos.... )

File: lib/sensorLayouts/index.js [hunk 1]
```diff
@@ -7,6 +7,14 @@ const senseboxhome = require('./sensebox.home'),
...
   luftdaten_sds011_bmp180 = require('./luftdaten/sds011.bmp180'),
   luftdaten_sds011_bme280 = require('./luftdaten/sds011.bme280'),
+  luftdaten_pms1003 = require('./luftdaten/pms1003'),
+  luftdaten_pms1003_bme280 = require('./luftdaten/pms1003.bme280'),
+  luftdaten_pms3003 = require('./luftdaten/pms3003'),
+  luftdaten_pms3003_bme280 = require('./luftdaten/pms3003.bme280'),
+  luftdaten_pms5003 = require('./luftdaten/pms5003'),
+  luftdaten_pms5003_bme280 = require('./luftdaten/pms5003.bme280'),
+  luftdaten_pms7003 = require('./luftdaten/pms7003'),
+  luftdaten_pms8003_bme280 = require('./luftdaten/pms7003.bme280'),
```
Comment (ubergesundheit) [inline lib/sensorLayouts/index.js]: this should be `luftdaten_pms7003_bme280` instead of `luftdaten_pms8003_bme280`

**Comment by ubergesundheit** [comment]
I changed the pull request to merge into `development`. If you fix the one error in the `lib/sensorLayouts/index.js`, I'll merge!

File: lib/sensorLayouts/index.js (hunk 1 above)
Comment (Adorfer) [reply lib/sensorLayouts/index.js]: should be fixed now

**Comment by ubergesundheit** [comment]
Ok nice! Thanks for your contribution!

**Pull request merged**